hphp/runtime/vm/jit/relocation-arm.cpp

```diff
@@ -877,8 +874,8 @@ void adjustInstruction(RelocationInfo& rel, Instruction* instr,
     auto adjusted = rel.adjustedAddressAfter(target);
     if (adjusted) {
       *addr = adjusted;
-      __builtin___clear_cache(reinterpret_cast<char*>(addr),
-                              reinterpret_cast<char*>(addr) + 8);
+      auto const begin = reinterpret_cast<TCA>(addr);
+      DataBlock::syncDirect(begin, begin + 8);
```

Comment:
Should we use `2 * vixl::kInstructionSize` here and above where we're using `8` directly?